doc/install/ceph-deploy/quick-rgw.rst

```diff
@@ -53,6 +53,11 @@ Configuring the Ceph Object Gateway Instance
    .. note:: Ensure that you leave no whitespace between ``port=<port-number>``
       in the ``rgw_frontends`` key/value pair.
 
+   .. note:: In a deployment with multiple Ceph Object Gateways, the default
+      port can be set for *all* gateway instances by putting the rgw_frontends
+      configuration into the ``[global]`` or section or a ``[client]`` section
```

Comment:
i am inclined to add this setting to `[client]` only. `[global`` is consumed by all ceph applications. and please quote "rgw_frontends" with double backquotes. and might want to correct the typo of "or section or a".

Reply:
That sounds fine. Have corrected and amended in a new commit.